src/pages/layer-2/index.tsx

```diff
@@ -0,0 +1,630 @@
+            title={"Are all these networks safe?"}
+            eventCategory="l2_hub"
+            eventAction="expand"
+            eventName="are all these networks safe"
+            visible={true}
+          >
+            <div className="flex flex-col gap-8">
+              <p>
+                While generally designed with robust security features, their
+                safety depends on the underlying technology, smart contract
+                security, and <a href="">maturity of the network</a>.
+              </p>
+              <p>
+                Users should perform due diligence, starting with small
+                transactions and staying updated on developments to ensure
+                secure usage.
+              </p>
+            </div>
+          </ExpandableCard>
+          <ExpandableCard
+            title={
+              "Why can't Ethereum scale its own chain instead of relying on these networks?"
+            }
+            eventCategory="l2_hub"
+            eventAction="expand"
+            eventName="why can't ethereum scale its own chain instead of relying on these networks"
+            visible={true}
+          >
+            <p>
+              Ethereum can&apos;t easily scale its own main chain because it
+              needs to stay secure and decentralized. Making the main chain
+              faster could make it less secure and more centralized. Ethereum
+              networks help by processing transactions off the main chain and
+              then using the main chain for security, allowing Ethereum to
+              handle more transactions without losing security or
+              decentralization.
+            </p>
+          </ExpandableCard>
+          <ExpandableCard
+            title={"Why is there no 'official' Ethereum networks?"}
+            eventCategory="l2_hub"
+            eventAction="expand"
+            eventName="why is there no official ethereum networks"
+            visible={true}
+          >
+            <p>
+              Just as there is no &apos;official&apos; Ethereum client, there is
+              no &apos;official&apos; Ethereum layer 2. Ethereum is
+              permissionless - technically anyone can create a layer 2! Multiple
+              teams will implement their version of a layer 2, and the ecosystem
+              as a whole will benefit from a diversity of design approaches that
+              are optimized for different use cases. Much like we have multiple
+              Ethereum clients developed by multiple teams in order to have
+              diversity in the network, this too will be how layer 2s develop in
+              the future.
+            </p>
+          </ExpandableCard>
+        </div>
+      </div>
+
+      {/* TODO: Setup for translation */}
+      <div
+        id="layer-2-callout-cards"
+        className="flex w-full flex-col gap-16 px-8 py-8 md:flex-row"
+      >
+        <Callout
+          image={ExploreImage}
```

Comment:
<img width="762" alt="image" src="https://github.com/user-attachments/assets/d0ccfcd9-2f04-4ef9-b4f6-623ec5c44dfa">

This image gets cut off

Reply:
I see we're reusing this from the /networks/ hero.. I wonder if we have another option that isn't chopped at the top

Reply:
Its not that its cut off, its just how that image is. I think we could pick another image if we want, ill bring it up with Jakub.

Reply:
Right right, it works as a hero where it's in a box, but isn't great for this spot imo, cc @konopkja

Reply:
<img width="2000" alt="core" src="https://github.com/user-attachments/assets/498d6193-e68a-4413-9674-5f16bd438286">

Maybe this one would be ok for now, and we can always change later?

Reply:
hmm i guess its ok to change though will be a little inconsistent with the hero